pallets/attestation/src/migrations.rs

```diff
@@ -35,7 +35,11 @@ where
 {
 	fn on_runtime_upgrade() -> frame_support::weights::Weight {
 		log::info!("Attestation: Initiating migration");
-		if is_upgraded::<T>() {
+
+		let onchain_storage_version = Pallet::<T>::on_chain_storage_version();
```

Comment:
Can we have an else branch logging something about the version requirement not being met?

Reply:
Resolved in: [f46eb0c](https://github.com/KILTprotocol/kilt-node/pull/527/commits/f46eb0c67c54ba242710cba7da5ad5d49f6cdef4)

Reply:
Nice. Perhaps use the same value when you run the check and when you print the result. E.g., don't repeat `StorageVersion::new(1)`, but rather put it in a variable and use it everywhere in the function.